resources/api/__init__.py

```diff
@@ -0,0 +1,27 @@
+from .base import all_views
+from .resource import ResourceListViewSet, ResourceViewSet, PurposeViewSet
+from .reservation import ReservationViewSet
+from .unit import UnitViewSet
+from .search import TypeaheadViewSet
+
+from rest_framework import routers
+
+
+class RespaAPIRouter(routers.DefaultRouter):
+    def __init__(self):
+        super(RespaAPIRouter, self).__init__()
+        self.registered_api_views = set()
+        self.register_views()
+        self.register("typeahead", TypeaheadViewSet, base_name="typeahead")
```

Comment:
I'd prefer the endpoint is called `search`, so that we don't have to change the API when we implement the full-text search functionality.